lib/src/app/app.dart

```diff
@@ -16,25 +17,151 @@ class AppClient {
 
   AppClient(this._client);
 
+  /// Get the id of the user with the email provided
+  Future<String> getUserIDByEmail(String email) async {
```

Comment:
[nit] `ID` should be `Id` (see [Dart Style Guide](https://dart.dev/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words))